sup3r/bias/bias_calc.py

```diff
@@ -1304,7 +1356,19 @@ def __init__(self,
         self.sampling = sampling
         self.log_base = log_base
 
-        super().__init__(base_fps, bias_fps, *args, **kwargs)
+        super().__init__(base_fps=base_fps,
```

Comment:
The larger problem is that the CLI command creation relies on inspect for the init signature and *args/**kwargs show up directly in the signature without knowing what they are used for. Explicit is the best way to avoid this without doing some annoying implicit tracking of how args/kwargs get used.

Reply:
Yeah but isn't there something that can walk through the inheritance tree and find explicit args/kwargs? I would think inspect could do this but maybe not

Reply:
i dont think so because args/kwargs aren't required to go to the `super().__init__()` method. you'd have to write an interpreter/parser to scrape the code and track the usage of the args.

Reply:
Shall I close this conversation?

Reply:
Yeah i think we're all good!